### PR TITLE
FORTECH-78: Removed duplicate gallium variables

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Removed duplicate gallium variables

Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

File: styles/colors-gallium-night.less [hunk 1]
```diff
@@ -3,178 +3,56 @@
...
 // ---------------------------------------
 @agate-spotlight-color:                @agate-foreground-focus;
 @agate-spotlight-bg-color:             @agate-background-focus;
-@agate-spotlight-border-color:         transparent;
 @agate-spotlight-disabled-text-color:  fade(@agate-spotlight-color, 40%);
-@agate-spotlight-focus-bg-image:       none;
 
 // Button
 // ---------------------------------------
-@agate-button-color:          @agate-foreground;
```
Comment (daniel-stoian-lgp) [inline styles/colors-gallium-night.less:50]: not the same value

File: styles/colors-gallium-night.less [hunk 2]
```diff
@@ -3,178 +3,56 @@
...
-@agate-button-bg-color:       @agate-dark-gray;
-@agate-button-bg-image:       none;
-@agate-button-border-color:   transparent;
-@agate-button-shadow:         none;
 @agate-button-focus-color:    @agate-spotlight-color;
```
Comment (daniel-stoian-lgp) [inline styles/colors-gallium-night.less:24]: missed this one

**Comment by ybsung** [comment]
I refreshed the branch because of the conflict.

**Comment by Djspaceg** [comment]
I brought back just the two deleted core variables. It seemed wrong to not have them there, though there's no technical reason they must exist, since they're already defined as those colors. These variables at the top are a little special, so I wanted to keep discoverability of them high, rather than only existing in the base files like the other component-specific variables.

Also, I switched over the keyword colors to their corresponding agate defined color variables as a good practice. "black" doesn't match "@agate-black" so it was not switched.

**Pull request merged**